### PR TITLE
Filter preps GUI + some minor fixes

### PR DESCRIPTION
**Pull request by antgonza**
Final touch to filter prep tabs via JS:
filter-preps-GUI

### CODE REVIEW

File: qiita_ware/private_plugin.py [hunk 1]
```diff
@@ -402,9 +402,6 @@ def list_remote_files(job):
             job._set_error(traceback.format_exception(*exc_info()))
         else:
             job._set_status('success')
-        finally:
-            # making sure to always delete the key so Qiita never keeps it
```
Comment (wasade) [inline qiita_ware/private_plugin.py:406]: ?

**Review by wasade — COMMENTED** [review]
Minor question

File: qiita_ware/private_plugin.py (hunk 1 above)
Comment (antgonza) [reply qiita_ware/private_plugin.py:406]: This is the bug we found that was preventing listing files via scp for study 12294; basically `list_remote` already removes the key and this fails cause the file never exists when we reach this point.

Comment (wasade) [reply qiita_ware/private_plugin.py:406]: ah, thanks!

**Pull request merged**